### ISSUE TITLE
Sea Serpents Don't Drown When Stuck on Land

### ISSUE THREAD

**Issue by PlatinumFox** [Alex-the-666/Ice_and_Fire#1513]
serpents dont drown when they get stuck on land. they should start drowning or at least make an attempt to FLOP back to water.

**Comment by VanillaVanity7** [comment]
was going to make report, but I saw this already here so I'll jump on board with the same suggestion. Plus a screenshot, because it's just so bizarre for them to be so far inland. This dude jumped after me when I was on shore collecting shells & proceeded to chase me inland. When I finally got away he just killed a sheep instead & slid in to the water like it was totally normal. 

![why](https://user-images.githubusercontent.com/47676854/55646947-6dd14980-57aa-11e9-8b22-a3d4c7e0001e.png)

**Comment by Thethuthinnang** [comment]
I've seen the same behavior. A sea serpent came out of the water to chase a villager once and followed him fairly far inland until it killed him. It seemed only slightly slower than in water and didn't seem in a hurry to get back into water.  

It was fearsome! I hadn't seen them soar out of the water before, though I knew they could, so I just enjoyed the show. It was a steep cliff, too, so the jump it made was really spectacular. (I made sure to stop building too close to the ocean after that, though. lol)
